### PR TITLE
docs(suspensive.org): add contributors page

### PR DESCRIPTION
**Pull request by manudeli**
# Overview

I added contributors page for https://suspensive.org to promote contributors and maintainers.
We don't forget who suspensive could be made by

chrome-capture-2024-6-30 (5)

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (manudeli) [inline .github/CODEOWNERS:1]: What is CODEOWNER’s DRI? In my opinion, It is to strictly review all PRs to be merged into main and ensure that the library is distributed reliably to library users and that the development process flows smoothly.

Recently, @okinawaa and @minsoo-web have had their own DRIs, so I will remove them from CODEOWNERS, but I will remember you on the contributors page. In this repository, but we maintain maintain privileges as collaborators so that you can help with Suspensive yet.

When the company inquired with the Head of frontend, they responded that external maintainers (@gwansikk, @kangju2000) do not have CODEOWNERS privileges. This means that merging into main is only possible through @manudeli.

File: docs/suspensive.org/src/pages/docs/contributors.en.mdx
Comment (minsoo-web) [inline docs/suspensive.org/src/pages/docs/contributors.en.mdx:1]: This is a very minor comment, so please take it lightly.

I think the current purpose of the maintainer or contributor page is to **encourage contributions**, and if it's meant to be more encouraging, what if the page looked different based on **the size of the contribution**? 

The simplest UI direction I could think of was something like a bubble chart

example image
image

It's good enough as it is! Thank you!

**Review by minsoo-web — APPROVED** [review]

File: docs/suspensive.org/src/pages/docs/contributors.en.mdx
Comment (manudeli) [reply docs/suspensive.org/src/pages/docs/contributors.en.mdx:1]: https://github.com/toss/suspensive/issues/1012 I left this as issue! Cool idea....! Thanks! I'll care it after merging

**Review by kangju2000 — APPROVED** [review]
COOL!! 🙌

**Review by okinawaa — APPROVED** [review]
Thanks a lot 
I love

**Pull request merged**
